pkg/sync/v5/data/ConfigMap/openshift-azure-logging/fluentd-config.yaml

```diff
@@ -22,6 +22,17 @@ data:
         tag journald
       </rule>
     </match>
+    <filter journald>
+      @type record_transformer
+      enable_ruby true
+      <record>
+       #k8s_apiserver_apiserver-vqxg4_kube-service-catalog_72ce4d73-5224-11e9-98d0-000d3a196756_0
+       CONTAINER ${unless record["CONTAINER_NAME"].nil? ; record["CONTAINER_NAME"].split("_")[1] ; else "" ; end; }
+       POD ${unless record["CONTAINER_NAME"].nil?; record["CONTAINER_NAME"].split("_")[2] ; else "" ; end; }
```

Comment:
`else ""` needed if we dont want to see `Null` values in Geneva. an empty string value is a default for all other empty fields.

Reply:
Out of curiosity, why do we prefer an empty string to Null here?

Reply:
my preference is to see a unified view in UI, not some fields `""` some `Null` . this way fields are either populated or `""`
 view now is empty fields if field is not populated 
![image](https://user-images.githubusercontent.com/3225744/55896481-4a493d00-5bb6-11e9-9307-3eaad3cf9092.png)

I dont have a hard preference :man_shrugging:

Reply:
empty string is not the default, nil is - please fix

Reply:
![image](https://user-images.githubusercontent.com/3225744/55943323-4a8d1b00-5c3e-11e9-82f5-fdb0798cefcf.png)
nill is our new introduced values. <empty> is that any other components puts in into an empty field. I was just trying to follow data consistency